apisix/plugins/skywalking.lua

```diff
@@ -33,7 +33,7 @@ local metadata_schema = {
         service_instance_name = {
             type = "string",
             description = "User Service Instance Name",
-            default = "APISIX Instance Name",
+            default = core.utils.gethostname(),
```

Comment:
We can't use this way as the default value need to be constant

Reply:
let me see if there are other solutions

Reply:
Use placeholders to implement functionality